specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json

```diff
@@ -354,8 +354,7 @@
         },
         "collation": {
           "description": "Collation of the managed instance.",
-          "type": "string",
-          "readOnly": true
+          "type": "string"
```

Comment:
This is fully read-write. You need x-ms-mutability = read, create.

Reply:
In case you didn't see, @marstr requested changes because currently this swagger extension is not being used.